benchmarks/benchmarks.rb

```diff
@@ -3,18 +3,33 @@
 require "benchmark/ips"
 
 require "tempfile"
-require "memo_wise"
+
+github_memo_wise_path = Gem.loaded_specs["memo_wise"].full_gem_path
+
+GITHUB_MAIN = "GitHubMain"
```

Comment:
MINOR: In the results, I was wondering if this might be clearer as "MemoWise-GitHubMain" or otherwise indicating that it's the Github main branch of MemoWise?

Reply:
MINOR: Can we have a comment explaining that this string will be used both for the temp paths, as well as the reported results, for the version of MemoWise currently in Github main branch?